data/json/overmap/overmap_special/specials.json

```diff
@@ -4170,6 +4170,15 @@
     "occurrences": [ 10, 100 ],
     "flags": [ "CLASSIC", "WILDERNESS", "OVERMAP_UNIQUE" ]
   },
+  {
+    "type": "overmap_special",
+    "id": "outcrop",
+    "overmaps": [ { "point": [ 0, 0, 0 ], "overmap": "outcrop_north" } ],
+    "locations": [ "wilderness" ],
+    "city_distance": [ 30, -1 ],
+    "occurrences": [ 0, 50 ],
```

Comment:
50 max occurrences??

Reply:
Agreed, I've dropped it to ten to match the combination of the two cave types